tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -873,12 +873,6 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args, python_api_us
                 elif 'example_inputs' in args:
                     raise AssertionError("'example_inputs' argument is not recognized, maybe you meant to provide 'example_input'?")
 
-                if 'use_legacy_frontend' in args and args['use_legacy_frontend']:
```

Comment:
Please remove description of this functionality in documentation
https://github.com/openvinotoolkit/openvino/blob/eb178a753b5f0536b19efe597e805ef6634f16c6/docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md?plain=1#L52